error: method does not override or implement a method from a supertype

I'm am trying to get this up and running, but once I install this plugin, I can no longer run the application.  I get the following error:

```
node_modules\react-native-nordic-dfu\android\src\main\java\com\pilloxa\dfu\RNNordicDfuPackage.java:19: error: method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-nordic-dfu:compileReleaseJavaWithJavac FAILED
```

I am working in Windows 10 and testing on Android (IOS will be next).

--- a/android/src/main/java/com/pilloxa/dfu/RNNordicDfuPackage.java
+++ b/android/src/main/java/com/pilloxa/dfu/RNNordicDfuPackage.java
@@ -16,7 +16,6 @@ public class RNNordicDfuPackage implements ReactPackage {
       return Arrays.<NativeModule>asList(new RNNordicDfuModule(reactContext));
     }
 
-    @Override
     public List<Class<? extends JavaScriptModule>> createJSModules() {
       return Collections.emptyList();
     }

